routes/index.js

```diff
@@ -5,7 +5,6 @@ var crypter = require("../util/crypter.js")
 var validation = require("../public/scripts/validation.js");
 /* GET home page. */
 router.get("/", function(req, res, next) {
-  exports.hostname = req.get('host');
```

Comment:
Both of them works fine. The problem is in exporting it to that file. Tried but couldn't find the solution for that